src/core/2d_primitives.js

```diff
@@ -75,6 +75,12 @@ p5.prototype.arc = function(x, y, w, h, start, stop, mode) {
   for (var i = 0; i < args.length; ++i) {
     args[i] = arguments[i];
   }
+  // check with FES:validateNumericParameters
+  var validatePar = (this._validateNumParameters('arc', args, 6));
+  if (validatePar[0] === false) {
+    console.log(validatePar[3]);
+    throw new Error(validatePar[1]);
```

Comment:
This was mainly for simplifying the unit test but easily swapped out. I will remove comment-out for the log.console() in error_helpers.js and introduce back the function.